controllers/articles.controller.js

```diff
@@ -1,14 +1,25 @@
-const selectArticleById = require('../models/articles.models.js')
-
+const {
+  selectArticleById,
+  selectAllArticles,
+} = require("../models/articles.models.js");
 
+function getAllArticles(req, res, next) {
+  selectAllArticles().then((articles) => {
+    res.status(200).send({ articles });
+  }).catch((err) => 
+  {
+  console.log(err)
```

Comment:
you may want to invoke next with err instead of just to console.log it incase some 500 internal server error happens